subprojects/core/src/main/java/org/gradle/initialization/BuildClientSubscriptions.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.gradle.tooling.internal.provider;
```

Comment:
Why was this moved?

Reply:
It was moved to a place where `launcher`, `toolingApiBuilders` and `languageJava` could access it. Do you think there's a better project for it than in `core`? Do you think we should keep `tooling` in the package name?

Reply:
I'm not sure I understand why languageJava needs to access this now. Can you point me in the right direction?